package.json

```diff
@@ -20,6 +20,6 @@
   },
   "dependencies": {
     "atom-linter": "~3.3.0",
-    "htmlhint": "~0.9.12"
```

Comment:
Let's not change to a different htmlhint.  Add your custom changes directly to htmlhint.